### DISCUSSION TITLE
Please use the `docs` branch for documentation going forward

### DISCUSSION THREAD

**Discussion by MattReimer** [Riverscapes/riverscapes-tools#513] [So you know]
this is FYI for @joewheaton @jtgilbert and anyone else writing docs for `riverscapes-tools`

Starting today I've introduced a new branch called `docs` that will deploy directly to the website

The docs for docs (yes, it's doc-ception) is here: https://tools.riverscapes.xyz/developing/

## Q/A:

- **Why do we need another branch?**
    - it's getting really noisy on the master branch of this repo and starting to obscure the code changes.
    - Also, I've noticed some messy merge commits. We want to keep our `master` branch pristine and straight if possible but that takes a lot of work and not the kind of thing document writers should have to do. This seems like a nice compromise.

<img width="568" alt="Screen Shot 2021-12-03 at 1 22 54 PM" src="https://user-images.githubusercontent.com/1063391/144674595-25dcb70e-d4eb-4bb8-a017-b6265533a5d2.png">

- **Do I need to do pull-requests and feature branches to use `docs`?**
    - No but you can. Anything you commit and push to `docs` goes onto the website immediately so if there's a chunk of changes you want to make all at once spread across multiple commits or if you want someone to check your work first then pull requests have you covered.  

- **`origin/docs` moved and now I can't push**
    - Periodically I may flatten or rebase docs to catch up with the master branch. If this happens you will need to rebase on top of `docs` in order to push. Contact me if you need instructions on how that works. 

Anything else I missed feel free to ask. This is supposed to make things easier, not harder so let me know if I can make it better.

**Comment by jtgilbert** [comment]
Thanks @MattReimer , this sounds like a good plan.
